.reflame.config.jsonc

```diff
@@ -58,6 +59,94 @@
     "https://unpkg.com/@highlight-run/rrweb@0.9.27/dist/index.css",
     "/public.css",
   ],
+  "metas": [
+    {
+      "name": "msapplication-TileColor",
+      "content": "#5629c6",
+    },
+    {
+      "name": "title",
+      "content": "highlight.io | The Open Source, Fullstack Monitoring Platform"
+    },
+    {
+      "property": "og:type",
+      "content": "website"
+    },
+    {
+      "property": "og:url",
+      "content": "https://app.highlight.io/"
+    },
+    {
+      "property": "og:title",
+      "content": "highlight.io | The Open Source, Fullstack Monitoring Platform"
+    },
+    {
+      "property": "og:description",
+      "content": "Highlight gives your team full transparency into the errors, interactions, and performance metrics of your client-side application."
+    },
+    {
+      "property": "og:image",
+      "content": "/hero.png"
+    },
+    {
+      "property": "twitter:card",
+      "content": "/hero.png"
+    },
+    {
+      "property": "twitter:url",
+      "content": "https://app.highlight.io/"
+    },
+    {
+      "property": "twitter:title",
+      "content": "highlight.io | The Open Source, Fullstack Monitoring Platform"
+    },
+    {
+      "property": "twitter:description",
+      "content": "Highlight gives your team full transparency into the errors, interactions, and performance metrics of your client-side application."
+    },
+    {
+      "property": "twitter:image",
+      "content": "/hero.png"
+    },
+  ],
+  "links": [
+    {
+      "rel": "apple-touch-icon",
+      "sizes": "180x180",
+      "href": "/apple-touch-icon.png",
+    },
+    {
+      "rel": "apple-touch-icon",
+      "href": "/logo192.png",
+    },
+    {
+      "rel": "icon",
+      "sizes": "32x32",
+      "href": "/favicon-32x32.png",
+    },
+    {
+      "rel": "icon",
+      "sizes": "16x16",
+      "href": "/favicon-16x16.png",
+    },
+    {
+      "rel": "mask-icon",
+      "color": "#5629c6",
+      "href": "/safari-pinned-tab.svg",
+    },
+    {
+      "rel": "preconnect",
+      "href": "https://fonts.gstatic.com",
+      "crossorigin": true
+    }
+    // FIXME: this doesn't actually work yet
+    // Need to rewrite icon urls within the manifest
+    // Probably need a first class webmanifest config api instead
+    // {
+    //   "rel": "manifest",
+    //   "href": "/manifest.json"
+    // }
```

Comment:
Will follow up with a separate PR for this. Curious if this would be a blocker for going production?

Reply:
Nvm, implemented this earlier today and updated this PR directly: https://github.com/highlight/highlight/pull/7106/commits/8ce4b81f8487baafa9a731d38280ceb58eb8aba7

Preview is now installable. :)
![Screenshot 2023-11-08 at 7 39 37 PM](https://github.com/highlight/highlight/assets/6934200/48b64591-7b34-424b-ab73-6dfff8c9c448)